src/Symfony/Component/Security/Core/Resources/translations/security.lv.xlf

```diff
@@ -64,11 +64,11 @@
             </trans-unit>
             <trans-unit id="17">
                 <source>Too many failed login attempts, please try again later.</source>
-                <target>Pārāk daudz atteiktu ieejas mēģinājumu, lūdzu, mēģiniet vēlreiz vēlāk.</target>
+                <target>Pārāk daudz atteiktu autentifikācijas mēģinājumu, lūdzu, mēģiniet vēlreiz vēlāk.</target>
```

Comment:
Google Translate says that your proposed change means:

```
Too many rejected authentication attempts, please try again later
```

But the original meant:

```
Too many rejected login attempts, please try again later.
```

which is exactly like the original.

I'm not saying that you are wrong, but please double check that changing "login attempt" by "authentication attempt" is the right change here. Thanks.

Same for the change of _"Ieejas"_ by _"Autentifikācijas"_ in the other message.

Reply:
"ieejas" in this context is wrong word as you are not phisically going in somewhere. I chose "autentifikācijas" as it was used for "login" in other keys. Technically best word here for "login" is "pieslēgties" but then change must be made in more translation keys. I can do that later

Reply:
OK, so if you double checked this, then it's OK. Thanks for the explanation.